docs/pages/reference/networking.mdx

```diff
@@ -206,3 +215,33 @@ Read more in our [TLS Routing](../architecture/tls-routing.mdx) guide.
 
 </TabItem>
 </Tabs>
+
+### Agent ports
+
+For Teleport processes running agents, e.g., instances of the SSH Service,
+Kubernetes Service, and other services that protect resources in your
+infrastructure, there is no need to open ports on the instance to the public
+internet. 
+
+Teleport agents dial the Teleport Proxy Service to establish a reverse tunnel.
```

Comment:
Nit: I feel like this could be worded to more strongly express that it's because of this reverse tunnel that ports do not need to be opened/exposed.

Reply:
I've rearranged the paragraphs a bit to do this